ambry-api/src/main/java/com/github/ambry/quota/StorageUsageRefresher.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+import java.util.Map;
+
+
+/**
+ * An interface to initialize and refresh the storage usage for each container.
+ */
+public interface StorageUsageRefresher {
+
+  /**
+   * Return the current storage usage for each container. The returned map should be structured as such:
+   * The key of the map is the account id in string format and the value of the map is the storage usage of each
+   * container under this account.
+   * The container usage map's key is the container is in string format, and the value is storage usage of this container.
+   * @return The current storage usage for each container.
+   * @throws Exception
+   */
+  Map<String, Map<String, Long>> getContainerStorageUsage() throws Exception;
```

Comment:
This is more like Account Storage Usage. Because you are basically getting quota usage for all containers of an account.

Reply:
It actually returns container storage usage for all accounts. The outer map's key is the account id, so it covers all the accounts and containers. The reason why this is "getContainerStorageUsage" not "getAccountStorageUsage" is to emphasize the storage usage is per container, not per account.